unit-tests/src/test/java/org/eclipse/collections/impl/bag/mutable/MutableBagTestCase.java

```diff
@@ -212,14 +212,15 @@ public void forEachWithOccurrences()
     public void collectWithOccurrences()
     {
         Bag<Integer> bag = this.newWith(3, 3, 3, 2, 2, 1);
-        Bag<ObjectIntPair<Integer>> actual =
+        Bag<ObjectIntPair<Integer>> actual1 =
                 bag.collectWithOccurrences(PrimitiveTuples::pair, Bags.mutable.empty());
-        Bag<ObjectIntPair<Integer>> expected =
+        Bag<ObjectIntPair<Integer>> expected1 =
                 Bags.immutable.with(
                         PrimitiveTuples.pair(Integer.valueOf(3), 3),
                         PrimitiveTuples.pair(Integer.valueOf(2), 2),
                         PrimitiveTuples.pair(Integer.valueOf(1), 1));
-        Assert.assertEquals(expected, actual);
+        Assert.assertEquals(expected1, actual1);
+        Assert.assertEquals(expected1, bag.collectWithOccurrences(PrimitiveTuples::pair));
```

Comment:
Why are these local variables renamed?

Reply:
It is my nitpick, if there are more than 1 expected and actual, then the latter ones are always named as expected2, so on and so forth. But we never go back to rename the first one. It creates confusion for me while reading the code as to was expected1 really deleted by mistake or never existed.